frontend/src/components/editor-page/editor-pane/linter/single-line-regex-linter.ts

```diff
@@ -52,11 +52,10 @@ export class SingleLineRegexLinter implements Linter {
   }
 
   private createDiagnostic(from: number, found: RegExpExecArray): Diagnostic {
-    const replacedText = this.replace(found[1])
-    return {
-      from: from,
-      to: from + found[0].length,
-      actions: [
+    let actions: Action[] = []
```

Comment:
Instead of using a mutable var I suggest to extract the actions-array logic into another function and use this in the object below.